static/files/forms-data.json

```diff
@@ -0,0 +1,534 @@
+{
+  "forms": {
+    "/aws": {
+      "templatePath": "/aws/index.html",
+      "formId": "1266",
+      "isModal": true,
+      "modalId": "contact-modal",
+      "formData": {
+        "title": "Test form",
+        "introText": "Intro text",
+        "formId": "1266",
+        "returnUrl": "/appliance/thank-you",
+        "product": ""
+      },
+      "fieldsets": [
+        {
+          "title": "About your company",
+          "id": "about-company",
+          "fields": [
+            {
+              "type": "text",
+              "id": "company",
+              "label": "Company name",
+              "isRequired": true
+            },
+            {
+              "type": "text",
+              "id": "job-title",
+              "label": "Job title",
+              "isRequired": true
+            }
+          ]
+        },
+        {
+          "title": "What would you like to talk to us about?",
+          "id": "comments",
+          "isRequired": true,
+          "fields": [
+            {
+              "type": "long-text",
+              "id": "comments"
+            }
+          ]
+        },
+        {
+          "title": "How many devices?",
+          "id": "how-many-machines",
+          "inputName": "how-many-machines-do-you-have",
+          "isRequired": true,
+          "fields": [
+            {
+              "type": "radio",
+              "id": "less-5-machines",
+              "value": "less than 5",
+              "label": "< 5 machines"
+            },
+            {
+              "type": "radio",
+              "id": "5-to-15-machines",
+              "value": "5 to 15 machines",
+              "label": "5 - 15 machines"
+            },
+            {
+              "type": "radio",
+              "id": "15-to-50-machines",
+              "value": "15 to 50 machines",
+              "label": "15 - 50 machines"
+            },
+            {
+              "type": "radio",
+              "id": "50-to-100-machines",
+              "value": "50 to 100 machines",
+              "label": "50 - 100 machines"
+            },
+            {
+              "type": "radio",
+              "id": "greater-than-100",
+              "value": "greater than 100",
+              "label": "> 100 machines"
+            }
+          ]
+        },
+        {
+          "title": "How should we get in touch?",
```

Comment:
I think you could add a check here like 'no comment for lead', as you don't want these being attached to the comments for lead. I believe all fields in the 'about you' sections have a 'name' attribute, meaning they are attached to the payload